tech-far-hub/src/components/resouces-card.tsx

```diff
@@ -0,0 +1,38 @@
+import { Card, CardHeader, CardBody, CardFooter } from "@trussworks/react-uswds";
+import { ColumnSizes } from "@trussworks/react-uswds/lib/components/grid/types";
+import { Link } from "gatsby";
+import * as React from "react";
```

Comment:
Super small nit, I've seen some libraries order their import statements. I kinda like a top down approach (although there's reason to follow this). 
- frameworks first
- then 3rd party libraries
- then an alphabetized list of components